components/proxy/conf/Caddyfile

```diff
@@ -255,7 +255,7 @@ https://*.*.{$GITPOD_DOMAIN} {
 	}
 
 	# foreign content origin should be decoupled from the workspace (port) origin but the workspace (port) prefix should be the path root for routing
-	@foreign_content header_regexp host Host ^(.*)(foreign).ws(-[a-z0-9]+)?.{$GITPOD_DOMAIN}
```

Comment:
we should replace all occurrences, see above `.ws(?P<location>-[a-z0-9]+)?`

I don't think actually `location` is used

Reply:
Hi Anton, 

I see 4 lines and 6 occurrences including `[a-z0-9]`:
https://github.com/gitpod-io/gitpod/blob/main/components/proxy/conf/Caddyfile#L216
https://github.com/gitpod-io/gitpod/blob/main/components/proxy/conf/Caddyfile#L233
https://github.com/gitpod-io/gitpod/blob/main/components/proxy/conf/Caddyfile#L246
https://github.com/gitpod-io/gitpod/blob/main/components/proxy/conf/Caddyfile#L258

I am not sure what these are doing:
* `?P<workspaceID>`
* `?P<location>`

But will plan to take a peek tomorrow. Thanks a lot for the heads up! I'll let you know if I have any questions.

Reply:
Hi @akosyakov , got it, thank you very much for explaining, good to know. :)